game/templates/game/lesson_detail.html

```
@@ -1,433 +1,295 @@
-            Next Example
-        </button>
-    </div>
-
-    {{ board_examples|json_script:"board-examples-data" }}
 
-    {% endif %}
-
-    {% if lesson_steps %}
-
-    <div class="lesson-practice">
+            <button id="next-example-btn" class="complete-btn">Next Example</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)